app/services/statements/statements_query.rb

```diff
@@ -0,0 +1,28 @@
+module Statements
+  class StatementsQuery
+    def initialize(lead_provider:, params: {})
```

Comment:
Should definitely be a list of keyword arguments here, it's self-documenting. To know which params are allowed would mean reading the code.

Reply:
Also, it might be worth thinking about how we build/structure these queries, i.e., do we:

* allow loads of params/keyword arguments at initialization and just have a single method called `#statements` that returns all i.e., `Statements::StatementsQuery.new(lead_provider:, cohorts:, since: some_date)`
* make it chainable so it's something more like `Statements::StatementsQuery.new(lead_provider:).in_cohorts(cohorts).since(some_date).all`

Ultimately you can build the same queries with both, but I suspect the second implementation will be easier o the eye.